### PR TITLE
Enable 2024.3.0

### PR DESCRIPTION
**Pull request by artanokhov**

### CODE REVIEW

**Comment by artanokhov** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -52,6 +53,7 @@ requirements:
     - ccache
     - libprotobuf
     - flatbuffers
+    - gflags  # [linux64]
```
Comment (mryzhov) [inline recipe/meta.yaml:56]: why only linux?

Comment (artanokhov) [reply recipe/meta.yaml:56]: Because only linux failed last time

**Comment by mryzhov** [comment]
@conda-forge-admin, please rerender

**Comment by artanokhov** [comment]
@conda-forge-admin, please rerender

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**